clowdapp.yaml

```diff
@@ -423,7 +421,7 @@ parameters:
   value: 50m
 - name: REDIS_SSL
   description: 'Whether to use secured connection to Redis. Use string values of true or false'
-  value: "true"
+  value: "false"
```

Comment:
@vkrizan wanted this true by default. We should probably use the resourceTemplate or cicd-common definition to define it as false, like we did here: https://gitlab.cee.redhat.com/insights-platform/cicd-common/-/blob/master/bonfire_configs/ephemeral_apps.yaml#L35.

Reply:
The default *true* value is the reason why the client doesn't connect. we need it to be *false* as Clowder does not provide SSL routes for any service, that's handled by Istio . I would like to hear @vkrizan opinion here.

Reply:
Yes please if possible use safe default (`"true"`). Environment where it needs to be turned off can override the env var. For Istio case there would be an exception for Redis' port number.
/cc @lphiri

Reply:
I'm just saying, in all Clowder templates, we'll have a default `true` value, and we will have to force it to `false` in every place we need it. that sounds ... weird. All clowder-managed environments need this variable set to false is what I mean

Reply:
That's exactly the point, to disable SSL/security knowingly. We can set the default on ci-cd common configuration.